### ISSUE TITLE
[Secondary Toolbar]: Screen reader does not announce the selected/unselected state information for the buttons

### ISSUE THREAD

**Issue by seahawksq** [mozilla/pdf.js#14526]
If screen reader does not announce selected/unselected states for the buttons present under secondary toolbar then it would be difficult for the AT users who are screen reader dependent to understand what actions should be performed further related to that control.
![image](https://user-images.githubusercontent.com/25859586/152270358-d9f46359-97a3-46c8-a6b7-0970a1377bb9.png)

Configuration:
- Web browser and its version: Chrome Version 97.0.4692.71 (Official Build) (64-bit) /Edge Version 97.0.1072.69 (Official build) (64-bit)
- Operating system and its version: 21H2(OS build 22000.376)
- PDF.js version: 2.13.107
- Is a browser extension: No

Steps to reproduce the problem:
1. Navigate to https://mozilla.github.io/pdf.js/web/viewer.html
2. Open secondary toolbar by clicking Tools button.
3. Navigate to "Text Selection Tool" or "Hand Tool"

What is the expected behavior? (add screenshot)
Screen reader should announce the selected/unselected state information for the buttons present under the "Tools".

Screen Reader Tested:
 Chromium+Narrator and Chrome+NVDA.
